content/en/docs/languages/go/instrumentation.md

```diff
@@ -876,7 +875,159 @@ meterProvider := metric.NewMeterProvider(
 
 ## Logs
 
-The logs API is currently unstable, documentation TBA.
+Logs are distinct from metrics and traces in that **there is no user-facing
+OpenTelemetry logs API**. Instead, there is tooling to bridge logs from existing
+popular log packages (such as slog, logrus, zap, logr) into the OpenTelemetry
+ecosystem. For rationale behind this design decision, see
+[Logging specification](/docs/specs/otel/logs/).
+
+The two typical workflows discussed below each cater to different application
+requirements.
+
+### Direct-to-Collector (Experimental)
```

Comment:
Btw, once this section looses its experimental status, the hash IDs to the section will become invalid. I have two alternative suggestions:

1. Move the status mention into the section prose, and then you can link to the spec definition of the term, for example:

    ```markdown
    ### Direct-to-Collector
    
    **Status**: [Experimental](/docs/specs/otel/document-status/)
    ```

2. Add an explicit has ID (**here and for other headings**), for example: `{#direct-to-collector}`